client/web/compose/src/components/PageBlocks/RecordListConfigurator.vue

```diff
@@ -610,6 +611,22 @@
           </b-row>
 
           <b-row>
+            <b-col
+              v-if="!options.hideAddButton"
+              cols="12"
+              md="6"
+            >
+              <b-form-group
+                :label="$t('recordList.record.addRecordOptions')"
+                label-class="text-primary"
+              >
+                <b-form-select
+                  v-model="options.addRecordDisplayOption"
+                  :options="recordDisplayOptions"
```

Comment:
options should be different, since we're not opening a record here but creating it.